### PR TITLE
Issue #200: Edit default config for checkstyle tester to produce no violations

### PR DESCRIPTION
**Pull request by MEZk**
#200

It default config is not changed we will get the following diff report:

image

Sorry, but my IDEA continues reformatting blank lines with tabs even if I disable code formatting before a commit.

### CODE REVIEW

**Comment by romani** [comment]
Appveyor is restarted.

Please use plain text editor to remove tabs from file.

**Comment by romani** [comment]
build was restarted on master and now it is failing

**Comment by romani** [comment]
smth wrong with parsing of property file.
@rnveach , can you run this on your Windows to get us understanding what is going on ...

**Comment by rnveach** [comment]
@romani I believe the issue is the blank line isn't being skipped and is treated as a repository.

When I add `echo $line|hexdump -v` to print the line, I get the following results:
````
0000000 0a0d
0000002
````

**Comment by rnveach** [comment]
There is also a version difference in cygwin in build history:
bad: `Starting cygwin install, version 2.877`
good: `Starting cygwin install, version 2.874`

**Comment by romani** [comment]
appveyor config was updated - c77e408

@MEZk , please rebase on latest code to pass CIs

**Comment by MEZk** [comment]
@romani 
Done.

> Please use plain text editor to remove tabs from file.

Git on my laptop is configured to automatically replace tabs with spaces and also to remove blank lines which contain only tabs. Are we OK with tabs replacement in my_check.xml?

File: checkstyle-tester/my_check.xml [hunk 1]
```diff
@@ -21,9 +21,11 @@
     <module name="TreeWalker">
          <!-- Example of sevntu.checkstyle Check usage -->
          <!-- <module name="NestedSwitchCheck"/> -->
-         
+
          <!-- Example of checkstyle Check usage -->
-         <module name="AbstractClassName"/>
+        <module name="ThrowsCount">
```
Comment (romani) [inline checkstyle-tester/my_check.xml]: space is missed for indentation

File: checkstyle-tester/README.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ The script receives the following set of command line arguments:
 
 **patchConfig** (pc) - path to the patch checkstyle config file. It will be applied to patch branch (required if baseConfig is specified);
 
-**config** (c) - path to the checkstyle config file. It will be applied to base and patch branches (required if baseConfig and patchConfig are not secified);
+**config** (c) - path to the checkstyle config file. It will be applied to base and patch branches (required if baseConfig and patchConfig are not secified). The defalt config should be changed in order to be appropriate for your use purposes;
```
Comment (rnveach) [inline checkstyle-tester/README.md]: `defalt` -> `default`

File: checkstyle-tester/my_check.xml (hunk 1 above)
Comment (MEZk) [inline checkstyle-tester/my_check.xml]: No, comments above have incorrect indentation level, not the ``` <module name="ThrowsCount">```. I fixed the indentation level of comments.

File: checkstyle-tester/README.md (hunk 2 above)
Comment (MEZk) [inline checkstyle-tester/README.md]: Thanks. Done.

File: checkstyle-tester/my_check.xml (hunk 1 above)
Comment (MEZk) [inline checkstyle-tester/my_check.xml]: Fixed.

**Pull request merged**